validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/DepositProvider.java

```diff
@@ -70,6 +79,10 @@ public synchronized void onFinalizedCheckpoint(FinalizedCheckpointEvent event) {
     depositNavigableMap.headMap(finalizedState.getEth1_deposit_index()).clear();
   }
 
+  // It's sad that we have to do this. Is there any other way?
```

Comment:
Probably don't need this comment. :). Come to think of it though, we could maybe use a `default` method on the interface.  Not really sure what `EventChannel` would think of that but it would probably work.

Reply:
Haha had forgotten to delete that after our chat. Deleted now.